kedro-telemetry/kedro_telemetry/plugin.py

```diff
@@ -149,28 +149,37 @@ def _generate_new_uuid(full_path: str) -> str:
         return ""
 
 
-class KedroTelemetryCLIHooks:
+class KedroTelemetryHook:
     """Hook to send CLI command data to Heap"""
 
+    def __init__(self):
+        self.consent = None
+        self._sent = False
```

Comment:
Why is this private and the rest public?

Reply:
They should be all internal.

Reply:
@DimedS does this still need to be updated?

Reply:
thanks, fixed to private!